library/convertigo

```diff
@@ -1,11 +1,11 @@
 Maintainers: Nicolas Albert <nicolasa@convertigo.com> (@nicolas-albert), Olivier Picciotto <olivier.picciotto@convertigo.com> (@opicciotto)
 GitRepo: https://github.com/convertigo/docker
-GitCommit: 59af509cbad0944fbbf2ad3cdb103d24e330cbec
+GitCommit: 24aac18434c36052877fd3e083db0e671e8bd742
 
 Tags: 7.5.7, 7.5, latest
-Architectures: amd64
+Architectures: amd64, arm64v8
```

Comment:
Unfortunately neither of the tomcat images you are `FROM` support `arm64v8` (`tomcat:7-jdk8-openjdk` and `tomcat:7-jdk8-openjdk-slim`). You'll need to use a different `FROM` like `tomcat:8.5-jdk11-openjdk-slim` if you want `arm64v8`. This is currently some of the ones based on jdk11 (see the [`tomcat` library file](https://github.com/docker-library/official-images/blob/7c850bb96cce7db58b06639d8c9caba2670b5915/library/tomcat))